cmd/gardenlet/app/app.go

```diff
@@ -80,8 +90,12 @@ import (
 	thirdpartyapiutil "github.com/gardener/gardener/third_party/controller-runtime/pkg/apiutil"
 )
 
-// Name is a const for the name of this component.
-const Name = "gardenlet"
+const (
+	// Name is a const for the name of this component.
+	Name = "gardenlet"
+	// TempGRMConfigMapName is the name is the configMap that is temporarily deployed for DWD prober
+	TempGRMConfigMapName = "gardener-resource-manager-dwd"
```

Comment:
Remove this variable. You only use it in one place. Just inline the value there.